config/src/main/java/com/yahoo/config/subscription/impl/JRTConfigRequester.java

```diff
@@ -126,17 +127,17 @@ public void handleRequestDone(Request req) {
     }
 
     private void doHandle(JRTConfigSubscription<ConfigInstance> sub, JRTClientConfigRequest jrtReq, Connection connection) {
-        if (sub.getState() == ConfigSubscription.State.CLOSED) return; // Avoid error messages etc. after closing
         boolean validResponse = jrtReq.validateResponse();
+        if (log.isLoggable(LogLevel.DEBUG)) {
```

Comment:
For the future, you can use the log method that takes a supplier. Then the log message will only be created of the log level is enabled:

```
log.log(DEBUG, () -> msg)
```